doc/source/reference/series.rst

```diff
@@ -214,11 +214,17 @@ Missing data handling
 .. autosummary::
    :toctree: api/
 
-   Series.isna
-   Series.notna
+   Series.backfill
+   Series.bfill
    Series.dropna
+   Series.ffill
    Series.fillna
    Series.interpolate
+   Series.isna
```

Comment:
hmm funny we don't have Series.replace here

Reply:
added